### ISSUE TITLE
[npm link] react-admin is linking as react-admin-lerna

### ISSUE THREAD

**Issue by oksuz** [marmelab/react-admin#2469]
**What you were expecting:**
I added piece of code to react-admin source, and linked as described on documentation but i can't start because:
`npm link ../react-admin` links as `my_project/node_modules/react-admin-lerna`

**What happened instead:**
react-admin should link as react-admin

**Steps to reproduce:**
https://marmelab.com/react-admin/index.html 

`If you are using react-admin as a dependency, and if you want to try and hack it, here is the advised process` section.

**Environment**

* React-admin version: 2.4.0
* React version: 16.6.0
* Browser: chrome
* Stack trace (in case of a JS error): no trace

**Comment by djhi** [comment]
Until we fix the documentation, you can try:

```sh
npm link ../react-admin/packages/react-admin
```

**Comment by oksuz** [comment]
@djhi ok thank you.

**Comment by oksuz** [comment]
@djhi hello again, `make watch` isn't work now. I guess watch script moved  from `Makefile` to https://github.com/marmelab/react-admin/blob/master/packages/react-admin/package.json#L26 here.

I tried `npm run watch` command on path `packages/react-admin`  after linking, my project doesn't compile. I got this error `Module not found: Can't resolve 'react-admin' in '/Users/oksuz/Developer/projects/my-admin-project/src'`

I guess watch command is broke and then I found old watch script from commit history, it is
`NODE_ENV=production ../../node_modules/.bin/babel ./src -d lib --ignore spec.js,test.js --watch`.
I ran this command on path `packages/react-admin`. It succeed.  My project woke up but react-admin code changes didn't apply. 

react-admin npm linking instructions looking like wrong.
